docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -116,6 +119,7 @@ Example:
 |--------	|--------------|:-------:|:--------:|
 | `namespace` | Namespace to deploy the Prometheus Operator. | `monitoring` | false |
 | `grafana.admin_password` | Password for `admin` user in Grafana. If not provided it is auto generated and stored in secret `prometheus-operator-grafana`.  | - | false |
+| `grafana.secret_env` | Sensitive environment variables passed to Grafana pod and stored as secret. | - | false |
```

Comment:
Maybe link to https://grafana.com/docs/grafana/latest/installation/configuration/#configure-with-environment-variables?

Reply:
Added additional line. Even though this will be covered in a doc.